packages/models-library/src/models_library/basic_types.py

```diff
@@ -21,6 +21,14 @@ class PositiveDecimal(ConstrainedDecimal):
     gt = 0
 
 
+class BoundedPositiveDecimal(ConstrainedDecimal):
+    # NOTE: upper limit to avoid https://github.com/ITISFoundation/appmotion-exchange/issues/2
+    # NOTE: do not contraint in decimal places. Too strong validation error rather Decimal.quantize
+    # before passing the value
+    gt = 0
+    lt = 1e6
```

Comment:
thought: it is a bit weird that bounded = 1e6. I would expect somehow a way to set it up. Maybe it could be renamed to something less generic than "bounded"? such as BoundedTransactionPositiveDecimal.